### DISCUSSION TITLE
Use polarimetric SAR products

### DISCUSSION THREAD

**Discussion by impresionista** [sertit/eoreader#44] [Q&A] [answered]
Hi there! I'm working with a team to use sar quadpol data, applying some polarimetric operations to use Alpha, Anisotropy and Entropy. 

We have a GPT graph to obtain this bands.
Is there a way to apply this graph and load the results as bands?

**Comment by remi-braun** [comment, marked as answer]
Hello,

For now it is not possible and my knowledge in this field is really thin 😅

Would it work for non quadpol data ? Because if only one satellite handles this it is a bit against the purpose of the band mapping of EOReader.

In your opinion, should EOReader load these bands as any other ones ?

The main issue I think is that EOReader doesn't handle custom bands for now (and I don't know if it would).
Moreover, I don't think we really need this feature for our CEMS everyday work (but I will check with some coworkers that might be interested)

Maybe the easiest way of loading these bands for now is to embed some eoreader code in yours (see how we use the GPT graphs in EOReader code, it is in the sar_products file, i.e. the preprocess function)

If you need any help I am back at work on Monday so feel free to ask 😉
